setup.cfg

```diff
@@ -23,4 +23,4 @@ addopts = --cov mwclient test
 [flake8]
 max-line-length = 90
 ignore = 
-	W503  # Line break before binary operator
```

Comment:
According to https://flake8.pycqa.org/en/latest/user/configuration.html, inline comments are not supported, but they can be added in a separate line. Can you do that, please? I'd prefer if we preserved the comment.

Reply:
done.